tests/test_email/test_email_builder_render_application.py

```diff
@@ -41,7 +32,10 @@ def application() -> Application:
 
 
 @pytest.mark.parametrize("language", ["fi", "en", "sv", None])
+@patch_method(EmailTemplateValidator.render_template)
 def test_email_builder__get_subject(language, email_template, application):
+    EmailTemplateValidator.render_template.return_value = f"HTML content { language or "fi" }"
```

Comment:
`"HTML content { language or "fi" }"`, does ruff go against PEP8 by adding whitespace to f-string values?

Reply:
You mean between adding a space between `{` and `language`?
Doesn't look like it. I removed the whitespace and ruff isn't formatting it back so it must have been me (Or maybe I can blame copilot 😄)